search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/GenerativeQAParamExtBuilder.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2023 Aryn
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.opensearch.searchpipelines.questionanswering.generative;
```

Comment:
Believe it or not, we debated this quite a bit.  We are sticking with "pipeline" instead of "processor" because of the convention used in OpenSearch. e.g. https://github.com/opensearch-project/OpenSearch/blob/dc4ef8f4ef466c3509cc92a2fd52672d0e132008/modules/search-pipeline-common/src/main/java/org/opensearch/search/pipeline/common/RenameFieldResponseProcessor.java#L9